app/api/service.py

```diff
@@ -0,0 +1,28 @@
+from flask_rest_jsonapi import ResourceDetail, ResourceList
+
+from app.api.bootstrap import api
+from app.api.schema.services import ServiceSchema
+from app.models import db
+from app.models.service import Service
+
+
+class ServiceList(ResourceList):
+    """
+    List all services i.e. microlocation, session, speaker, track, sponsor
+    """
+    decorators = (api.has_permission('is_admin', methods="GET"),)
+    methods = ['GET']
```

Comment:
Same question, Is POST request not allowed on this ? If not, then it's fine but if it is then add it.

Reply:
No post required